java-dynamic-sqs-listener-api/src/main/java/com/jashmore/sqs/QueueProperties.java

```diff
@@ -0,0 +1,18 @@
+package com.jashmore.sqs;
+
+import lombok.Builder;
+import lombok.Value;
+import lombok.experimental.NonFinal;
+
+/**
+ * Contains information about the queue that will be listened against.
+ */
+@Value
+@NonFinal
+@Builder
+public class QueueProperties {
```

Comment:
Should this be an interface? `queueUrl` is not needed anywhere in API. I think `queueUrl` is basic and common enough to be in here, just a thought.

Reply:
It originally was an interface but I thought that it was a bit unnecessary as I wouldn't ever really need a new implementation of this (but maybe I will in the future?) I don't care either way